src/faq/array/flattening-array.ts

```diff
@@ -0,0 +1,7 @@
+import { pipe, Effect } from "effect"
+
+// Effect.Effect<never, never, Array<number>>
```

Comment:
should `// $ExpectType Effect<never, never, number[]>`

Reply:
I would also move these two files to `/src/essentials/pipeline`

Reply:
Done 👍